Language_Project/urls.py

```diff
@@ -4,4 +4,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/v1/profiles/', include('profiles.urls')),
+    path('api/v2/platforms/', include('platforms.urls')),
```

Comment:
it's still v1 .
1. We use REST approach 
2. use DRF with the same version
3. payload is the same
4. no clients - no one use previous implementation
no reason to mark it as a new version

Reply:
good naming. well done